src/fsearch_file_utils.c

```diff
@@ -207,6 +208,7 @@ launch_context_new(GAppLaunchContext *app_launch_context,
     launch_ctx->launch_desktop_files = launch_desktop_files;
     launch_ctx->launch_uris_ctx_queue = g_queue_new();
     launch_ctx->path_queue = g_queue_new();
+    launch_ctx->recents_manager = gtk_recent_manager_get_default();
```

Comment:
we don't need to free this in `launch_context_free()` below since it's the default one managed by GTK